docs/use-cases/experiments.md

```diff
@@ -0,0 +1,316 @@
+| Purpose                  | Compare two or more variants of a single element               | Simultaneously test multiple elements and variations                                           |
+| Variants                 | Two or more variants (Control and Treatment)                   | Multiple variants for each element being tested                                                |
+| Scope                    | Focuses on one element at a time                               | Tests combinations of elements and their variations                                            |
+| Complexity               | Relatively simpler to set up and analyze                       | and analyze	More complex to set up and analyze                                                 |
+| Statistical Significance | Typically requires fewer samples to achieve significance       | Requires larger sample sizes to achieve significance                                           |
+| Insights                 | Provides insights into the impact of individual changes        | Provides insights into the interaction between changes                                         |
+| Test Duration            | Generally shorter duration                                     | Often requires longer duration to obtain reliable results                                      |
+| Examples                 | Ideal for testing isolated changes, UI tweaks, copy variations | Useful for testing larger-scale changes, page redesigns, interaction between multiple elements |
+
+## Your application
+
+For this guide, let's say your application consists of a landing page containing these elements:
+
+- **Hero section**: The main section of the landing page, which includes:
+  - headline
+  - subheading, and
+  - call-to-action (CTA) button
+
+We now want to run both A/B Tests and Multivariate Tests using Featurevisor.
+
+## A/B Test on CTA button
+
+Let's say we want to run an A/B Test on the CTA button in the Hero section of your landing page.
+
+The two variations for a simple A/B test experiment would be:
+
+- **control**: The original CTA button with the text "Sign up"
+- **treatment**: The new CTA button with the text "Get started"
+
+We can express that in Featurevisor as follows:
+
+```yml
+# features/ctaButton.yml
+description: CTA button
+tags:
+  - all
+
+bucketBy: deviceId
+
+defaultVariation: control
+
+variations:
+  - value: control
+    description: Original CTA button
+    weight: 50
+
+  - value: treatment
+    description: New CTA button that we want to test
+    weight: 50
+
+environments:
+  production:
+    rules:
+      - key: "1"
+        segments: "*" # everyone
+        percentage: 100 # 100% of the traffic
+```
+
+We just set up our first A/B test experiment that is:
+
+- rolled out to 100% of our traffic to everyone
+- with a 50/50 split between the `control` and `treatment` variations
+- to be bucketed against `deviceId` attribute (since we don't have the user logged in yet)
```

Comment:
Maybe we want to help the user here a bit more by giving them more context around the important of the ID especially around persistent bucketing and the importance of this in testing

Reply:
added a new callout section mentioning the importance of consistent bucketing with links to further guides.